### DISCUSSION TITLE
Implementation in Android

### DISCUSSION THREAD

**Discussion by Psijic** [mapsforge/vtm#1057] [Q&A] [answered]
Just implemented osmdroid within a couple of hours and chatbot help but can't find any manual how to implement Vtm in Android, use online maps and add marker clusters. Is there any? 
Got only this:
```
    implementation ("org.mapsforge:vtm:0.20.0")
    implementation ("com.caverock:androidsvg:1.4")
    implementation ("org.mapsforge:mapsforge-map-android:0.20.0")
```

**Comment by devemux86** [comment, marked as answer]
See the samples in [vtm-android-example](https://github.com/mapsforge/vtm/tree/master/vtm-android-example).

Also the [GettingStarted](https://github.com/mapsforge/vtm/blob/master/vtm-android-example/src/org/oscim/android/test/GettingStarted.java) and [integration guide](https://github.com/mapsforge/vtm/blob/master/docs/Integration.md).
